python_modules/libraries/dagster-ssh/dagster_ssh/resources.py

```diff
@@ -261,6 +292,4 @@ def sftp_put(self, remote_filepath, local_filepath, confirm=True):
     }
 )
 def ssh_resource(init_context):
-    args = init_context.resource_config
-    args = merge_dicts(init_context.resource_config, {"logger": init_context.log})
-    return SSHResource(**args)
+    return SSHResource.from_resource_context(init_context)
```

Comment:
is the config for the function resource different from the ConfigurableResource so we cant do `SSHResource.to_config_schema()`